packages/page-alerts/src/js/react.js

```diff
@@ -21,9 +21,9 @@ import classNames from 'classnames';
 const PageAlert = ( { variant = null, children } ) => {
```

Comment:
@dominikwilkowski @elisechant Would love some feedback on this component if you have the time :)